docs/podman-create.1.md

```diff
@@ -209,6 +209,13 @@ inside of the container.
    Expose a port, or a range of ports (e.g. --expose=3300-3310) to set up port redirection
    on the host system.
 
+**--gidmap**=map
+   GID map for the user namespace.  Using this flag will run the container with user namespace enabled, unless the --userns=host flag is turned on.
+
+   The following example maps uids 0-2000 in the container to the uids 30000-31999 on the host and gids 0-2000 in the container to the gids 30000-31999 on the host.
```

Comment:
Throw the example in the examples section at the bottom of the manpage

Reply:
hm.. it seems there is no EXAMPLES section in the file